janus_core/calculations/single_point.py

```diff
@@ -236,15 +237,11 @@ def _get_potential_energy(self) -> MaybeList[float]:
         MaybeList[float]
             Potential energy of structure(s).
         """
-        tag = f"{self.architecture}_energy"
-        if isinstance(self.struct, list):
+        if isinstance(self.struct, Sequence):
             energies = [struct.get_potential_energy() for struct in self.struct]
-            for struct, energy in zip(self.struct, energies):
-                struct.info[tag] = energy
             return energies
 
         energy = self.struct.get_potential_energy()
-        self.struct.info[tag] = energy
         return energy
 
     def _get_forces(self) -> MaybeList[ndarray]:
```

Comment:
Do these want to be `MaybeSequence` if we're allowing `Sequence`s?

Reply:
`Atoms` can be a sequence, but we still only return their forces as a list, so I think this is still accurate at the moment.